mappings/net/minecraft/entity/ai/brain/task/Task.mapping

```diff
@@ -0,0 +1,33 @@
+CLASS ajf net/minecraft/entity/ai/brain/task/Task
+	CLASS ajf$a Status
+	FIELD b timeLimit J
+	FIELD c minRunTime I
+	FIELD d maxRunTime I
+	METHOD a getRequiredMemoryState ()Ljava/util/Set;
+	METHOD a isTimeLimitExceeded (J)Z
+		ARG 1 time
+	METHOD a shouldRun (Lvc;Laif;)Z
+	METHOD a run (Lvc;Laif;J)V
+		ARG 1 world
+		ARG 2 entity
+		ARG 3 time
+	METHOD b tryStarting (Lvc;Laif;J)Z
+		ARG 1 world
+		ARG 2 entity
+		ARG 3 time
+	METHOD c tick (Lvc;Laif;J)Z
+		ARG 1 world
+		ARG 2 entity
+		ARG 3 time
+	METHOD d keepRunning (Lvc;Laif;J)V
```

Comment:
`shouldKeepRunning`?

Reply:
This is called to make the task continue running, if it returned true when `shouldKeepRunning` was called.

Reply:
Ah, I didn't notice `shouldKeepRunning` existed already.